lib/fastly_nsq.rb

```diff
@@ -26,19 +37,37 @@ def logger
       @logger ||= Logger.new(nil)
     end
 
+    ##
+    # Configuration for FastlyNsq
```

Comment:
Nice; we might be able to pull in some of the more in-depth examples in the readme into here.